dataactbroker/handlers/fileHandler.py

```diff
@@ -108,6 +109,9 @@ def submit(self,name,CreateCredentials):
                 # If filetype not included in request, and this is an update to an existing submission, skip it
                 if not safeDictionary.exists(fileType):
                     if existingSubmission:
+                        submission = self.jobManager.getSubmissionById(submissionId)
```

Comment:
Great idea to make sure the submission table timestamp is reset for a submission update. What do you think about doing that outside of the filetype loop to prevent potential multiple submission table updates?